integration-tests/snapshot-escape/__tests__/__snapshots__/snapshot_escape_regex.js.snap

```diff
@@ -1,9 +0,0 @@
-// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
Looks like this was added to the source by accident

Reply:
this usually happens if a test run is interrupted. I wonder if we should go through and add all the files we write and then remove from disk to `.gitignore` to avoid accidentally committing them